docs/outlook/online-meeting.md

```diff
@@ -0,0 +1,182 @@
+    }
+);
+
+function insertContosoMeeting(event) {
+    // Get HTML body from the client.
+    mailboxItem.body.getAsync("html",
+        { asyncContext: event },
+        function (getBodyResult) {
+            if (getBodyResult.status === Office.AsyncResultStatus.Succeeded) {
+                updateBody(getBodyResult.asyncContext, getBodyResult.value);
+            } else {
+                console.error("Failed to get HTML body.");
+                getBodyResult.asyncContext.completed({ allowEvent: false });
+            }
+        }
+    );
+}
+```
+
+This sample shows an implementation of the supporting function `updateBody` used in the previous example that appends the online meeting details to the current body of the meeting.
+
+```js
+function updateBody(event, existingBody) {
+    // Append new body to the existing body.
+    mailboxItem.body.setAsync(existingBody + newBody,
+        { asyncContext: event, coercionType: "html" },
+        function (setBodyResult) {
+            if (setBodyResult.status === Office.AsyncResultStatus.Succeeded) {
+                setBodyResult.asyncContext.completed({ allowEvent: true });
+            } else {
+                console.error("Failed to set HTML body.");
+                setBodyResult.asyncContext.completed({ allowEvent: false });
+            }
+        }
+    );
+}
+```
+
+### Testing and validation
+
+You can follow the usual guidance to [test and validate your add-in](testing-and-tips.md). After [sideloading](sideload-outlook-add-ins-for-testing.md) in Outlook on the web, Windows, or Mac, you should restart Outlook on your mobile device (Android is the only supported client for now) then verify on a new meeting screen that the Microsoft Teams or Skype toggle has been replaced with your own.
+
+#### Create meeting UI
+
+As a meeting organizer, you should see screens similar to the following when you create a meeting.
+
+[![screenshot of create meeting screen on Android - loading Contoso toggle](../images/outlook-android-create-online-meeting-load.png)](../images/outlook-android-create-online-meeting-load-expanded.png#lightbox) [![screenshot of create meeting screen on Android - Contoso toggle off](../images/outlook-android-create-online-meeting-off.png)](../images/outlook-android-create-online-meeting-off-expanded.png#lightbox) [![screenshot of create meeting screen on Android - Contoso toggle on](../images/outlook-android-create-online-meeting-on.png)](../images/outlook-android-create-online-meeting-on-expanded.png#lightbox)
+
+#### Join meeting UI
+
+As a meeting attendee, you should see a screen similar to the following when you view the meeting.
+
+[![screenshot of join meeting screen on Android](../images/outlook-android-join-online-meeting-view.png)](../images/outlook-android-join-online-meeting-view-expanded.png#lightbox)
+
+## Available APIs
+
+***TODO:*** Confirm set of APIs - will need to update ref docs for each of them - point to this article
```

Comment:
I assume this is a note to yourself? Triple *** is a horizontal rule in markdown!

Reply:
@lindalu-MSFT These TODOs are for updates to related docs (e.g. reference) and will be removed before publish. BTW, `***TODO:***` renders like this:
![image](https://user-images.githubusercontent.com/28541963/79013602-fbfab100-7b1d-11ea-8d7d-ae9e4d98c67c.png)